src/main/java/utility/PropertyManager.java

```diff
@@ -0,0 +1,31 @@
+package utility;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertyManager {
+    static String propertiesPath = "/Users/mlichota/IdeaProjects/APITesting/src/main/resources/app.poperties";
```

Comment:
also, the properties file is currently loaded every time a property is accessed. It's better to impliment a singleton pattern to load the properties file only once during the application's lifetime